plasTeX/Base/LaTeX/Math.py

```diff
@@ -84,15 +84,21 @@ def invoke(self, tex):
 
 class BeginMath(Command):
     macroName = '('
+    disableMath:bool = False
```

Comment:
The real package redefines `\(` and `\(` using a `\let` inside a `\begingroup`. If there is a way to redefine `(` so that it does nothing, but localised to only within `\ifthenelse`, that might be preferable. However, I'm not sure how to do that with plastex.